clean.ps1

```diff
@@ -38,4 +38,5 @@ if ($Help) {
     exit 0
 }
 
-git clean -dix -e .dotnet/ -e .tools/ -e src/SignalR/clients/ts/FunctionalTests/node_modules/ @GitArguments
+git clean -dix -e .dotnet/ -e .tools/ @GitArguments
+git checkout -- $(git ls-files -d)
```

Comment:
How is this different from `git reset --hard`?

Reply:
It only restores deleted files where the deletion hasn't been staged. It also doesn't undo modifications to checked-in files, unstage anything, and so on.